### PR TITLE
WINC-453: [hack] Extend OSDK run timeout to 5min

### PR DESCRIPTION
**Pull request by LorbusChris**
The tests have been failing a lot because
`operator-sdk run packagemanifests` has a default timeout of 2min which
was often missed. This commit extends the timeout to 5min to ensure the
command can return properly even when running on slower infrastructure.

With version 0.19.4 of the operator-sdk, the `run` command should now
return correct values, hence the condition and comment to explicitly
check for that have been removed.

Reverts ea557cb8ebb7a276e220af52fd9768ba10c6d16f

### CODE REVIEW

File: hack/common.sh
Comment (aravindhp) [inline hack/common.sh:43]: We are using SDK version 0.19.4. v0.16.9 must be the OLM version.

**Review by aravindhp — CHANGES_REQUESTED** [review]
Thanks for working on this @LorbusChris. Mostly LGTM. I suggest updating the commit message with why the cleanup is needed. Let's do a few retires and ensure this actually fixes the issue before merging.

File: hack/common.sh
Comment (LorbusChris) [reply hack/common.sh:43]: this is a typo, may bad! will push a fixup

**Comment by LorbusChris** [comment]
@aravindhp fixed the osdk version mentioned, and updated the commit message. Also updated the two backport PRs with this change

**Review by aravindhp — APPROVED** [review]
/lgtm
/hold

**Comment by aravindhp** [comment]
@LorbusChris I have approved. The hold is only to ensure that we see the retry doing what we expect before we merge.

**Comment by LorbusChris** [comment]
got OOMKilled again
/test aws-e2e-operator

**Comment by aravindhp** [comment]
/hold cancel
Saw the retry working [here](https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_windows-machine-config-operator/195/pull-ci-openshift-windows-machine-config-operator-master-aws-e2e-operator-with-custom-vxlan-port/1331630002470391808/artifacts/aws-e2e-operator-with-custom-vxlan-port/windows-e2e-operator-test-with-custom-vxlan-port/container-logs/test.log)

**Comment by LorbusChris** [comment]
I just realized that we've probably just been hitting the default timeout of 2 min on `operator-sdk run packagemanifests` which is configurable by a flag :facepalm: 
https://sdk.operatorframework.io/docs/cli/operator-sdk_run_packagemanifests/

I'll rework this...
/hold

File: hack/common.sh
Comment (aravindhp) [inline hack/common.sh:42]: Nice catch @LorbusChris. Hope it fixes the problem.

**Review by aravindhp — APPROVED** [review]
/lgtm

**Comment by aravindhp** [comment]
/hold cancel

Saw a couple of passing tests. Let's observe the signal from some follow up PRs.

**Pull request merged**
